vcluster/_partials/manage/deploy-changes.mdx

```diff
@@ -95,22 +95,36 @@ Replace:
 Learn more about [Cluster API Provider for vCluster](https://github.com/loft-sh/cluster-api-provider-vcluster).
 
 <Flow>
+<Step>
+
+   [Install](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) the `clusterctl` CLI.
```

Comment:
I would assume you already have this installed if you've already deployed a CAPI instance

Reply:
I considered this, but what if someone new is changing the vCluster version or vCluster yaml ? I am ensuring consistency in our steps. If this is an overkill, I am willing to let it slide.

Reply:
I wouldn't assume that anyone is new who would suddenly be managing a vcluster who doesn't have experience with CAPI.